.travis.yml

```diff
@@ -4,17 +4,61 @@ services:
   - docker
 node_js:
 - '8'
+
 env:
   global:
   - SAUCE_USERNAME=elastic-apm-base
   - SAUCE_ACCESS_KEY=4b6720f1-ecdc-448d-85be-ebf947e14507
   - APM_SERVER_PORT=8001
   - APM_SERVER_URL=http://localhost:8001
   - MODE=saucelabs
-  matrix:
-  - STACK_VERSION=6.5.0
-  - STACK_VERSION=6.6.0
-  - STACK_VERSION=7.0.0-alpha1-SNAPSHOT
+
+stages:
+  - lint
+  - test
+  - docs
+
+jobs:
+  include:
+    -
+      stage: docs
```

Comment:
would be happy to close my PR #167 with few changes here
1. Docs can be built only on master branch
2. cache: npm and not directory: node_modules since we run `npm ci`